containers/autoReply/AutoReplyForm/fields/StartDateField.js

```diff
@@ -18,7 +20,7 @@ const StartDateField = ({ value, onChange }) => {
                     defaultDate={new Date(value)}
                     setDefaultDate
                     onSelect={handleChange}
-                    format={moment.localeData().longDateFormat('L')}
+                    format={detectLongDateFormat(currentLocale)}
```

Comment:
Instead of format, it might be possible to pass:

```
toString={(date, format) => format(new Date(date), 'PP', { locale })}
```

Reply:
Thanks for the suggestion! I did not test much if this new solution is equivalent, but at any rate 'pikaday' will become eventually deprecated by our own library